File: plot_populations.m
clc %to clear the command
close all % to close eventual previous windows
clear all % to clear the workspace

FolderName = 'build/';

u1 = load([FolderName,'S.txt']);
u2 = load([FolderName,'Z.txt']);
u3 = load([FolderName,'R.txt']);
t = load([FolderName,'time.txt']);

plot(t, u1, 'b', t, u2, 'r', t, u3, 'g')
xlabel("Time");
ylabel("Individuals");

legend('S', 'Z', 'R')

File: plot_populations.py
from numpy import *
from pylab import *

S = loadtxt("S.txt")
Z = loadtxt("Z.txt")
R = loadtxt("R.txt")
t = loadtxt('time.txt')
plot(t,S,label="S")
plot(t,Z,label="Z")
plot(t,R,label="R")

xlabel(r'$Time$')
ylabel(r'$Individuals$')

legend(loc = "best")

#savefig("../../figs/SZR_fe_" + str(t.size-1) + ".png")
#savefig("../../figs/SZR_dirk_" + str(t.size-1) + ".png")

show()

